### ISSUE TITLE
Sentence Similarity not training

### ISSUE THREAD

**Issue by luisquintanilla** [dotnet/machinelearning-modelbuilder#2479]
**System Information (please complete the following information):**
 - Model Builder Version (available in Manage Extensions dialog): 17.14.3.2310605
 - Visual Studio Version: VS2022

**Describe the bug**
- On which step of the process did you run into an issue: Train
- Clear description of the problem: Model is supposedly training, GPU is not being used and there is not progress beyond Epoch 0.

**Expected behavior**
Model trains successfully

**Screenshots**

![image](https://user-images.githubusercontent.com/46974588/217303223-79e5bf02-b142-469e-b990-bc3f095f961c.png)

![image](https://user-images.githubusercontent.com/46974588/217303309-7ed92248-43f6-4451-8c24-28366f19bf23.png)

[Logs - SentenceSimilarity2-DZ9HQF.txt](https://github.com/dotnet/machinelearning-modelbuilder/files/10677300/SentenceSimilarity2-DZ9HQF.txt)

**Comment by IzzyHibbert** [comment]
Dear @luisquintanilla 

after a few days of investigation for a case I am having which is already open here :

https://github.com/dotnet/machinelearning-modelbuilder/issues/2428

I saw that you are experiencing something I am also (apparently) getting. It's just that I did not realize much that GPU training was not working at all. In my case it's an IMAGE CLASSIFICATION scenario.

Find here additional info from my side.

**My System Information :**

Visual Studio Version 17.3.5
Model Builder 16.14.0
GPU RTX 2060
Cuda release 10.1
CuDNN 7.6.4


![Screenshot 2023-02-08 104402](https://user-images.githubusercontent.com/45388237/217493714-f58e0c2c-9fce-4b92-a70c-361a366e6705.png)

After the training starts, there is no sign of errors (in log, in output, etc) and also - as in your case - no GPU load at all.
